build.gradle

```diff
@@ -163,7 +89,6 @@ allprojects {
     }
 
     detekt {
-        toolVersion = gradle.ext.detektVersion
```

Comment:
Warning (⚠️): Let's not remove this configuration unnecessarily. You could instead do the following:

```groovy
toolVersion = libs.versions.detekt.get()
```

Reply:
fixed

Reply:
FYI: It is usually a good practice to also include the commit that fixes a comment in your `fixed` comment. This will make it easier for a reviewer to actually review the fix much quicker. For example:

This is now done: 148586fcb3ddffc404e98cb123f43c91cfb2b15d